Sources/App/Controllers/Utils/Kleisli+Report.swift

```diff
@@ -42,3 +42,30 @@ extension Error {
         .init(description: "\(self)", code: "500")
     }
 }
+
+
+extension Kleisli where D: HasLogger {
+    func loggerM<E: Swift.Error>(_ f: @escaping (E) -> String,
```

Comment:
I think this method should be `log` and the one below `logInfo`, as they are actions. `logger` suggests you are retrieving the dependency, not performing the action.

Reply:
You're right, thanks. Updated